docs/compilers/CSharp/Compiler Breaking Changes - DotNet 6.md

```diff
@@ -6,4 +6,104 @@
     {
         if (o is null!) {} // error
     }
-    ```
\ No newline at end of file
+    ```
+
+2. In C# 10, lambda expressions and method groups with inferred type are implicitly convertible to `System.MulticastDelegate`, and bases classes and interfaces of `System.MulticastDelegate` including `object`,
+and lambda expressions are implicitly convertible to `System.Linq.Expressions.Expression` and `System.Linq.Expressions.LambdaExpression`.
```

Comment:
>lambda expressions [](http://example.com/codeflow?start=4&length=18)

I'd suggest saying "lambda expressions and method groups" here, and convert bullet 3 to be bullet 2c.